src/cli/probe.ts

```diff
@@ -0,0 +1,64 @@
+import { inputData, outputData } from '../lib/io';
+import { invokeUserFn, loadUserFn } from './lib/user-fn';
+import { verifyInputArray } from '../lib/verify';
+import {
+  standardCmdInputs,
+  standardCmdOutputs,
+  standardInputFileHelp,
+  standardOutputFileHelp,
+} from './lib/help';
+
+export async function main(argv: string[]): Promise<void> {
+  const { fn, details } = loadUserFn(argv, `record => transform(record)`);
+
+  const data = await inputData(argv);
+  verifyInputArray(data, 'map');
+
+  data.map((record: any) => {
+    const result = invokeUserFn(() => fn(record), details);
+    console.log(result);
```

Comment:
@Lukejkw I just realised this isn't going to work. 

Because Datakit relies on being able to output clean JSON data to the stdout you can't get away with console logging anything like this.

You could change it to `console.error`. Would that work for you?